### ISSUE TITLE
Is it possible to pin the timer to the top-right corner of entire editor?

### ISSUE THREAD

**Issue by nyngwang** [jackMort/pommodoro-clock.nvim#2]
It's easy to understand what I mean by a picture of the current status:

- So I can imagine that people who open multiple splits will have some of their buffers got covered by the timer.
- It will be cool if you can create some "follow mode" so that the timer will follow the current window.

<img src="https://user-images.githubusercontent.com/24765272/213000585-84682c62-631d-4f70-9724-0f42f52532bd.png" width="600">

**Comment by nyngwang** [comment]
> So I can imagine that people who open multiple splits will have some of their buffers got covered by the timer.

Can confirm that the first issue has been resolved by eb7a591.

**Comment by shakori999** [comment]
you just need to adjust the code 
```if M.current_state.popup == nil then
 M.current_state.popup = Popup({
   position = { row = 0, col = "100" } # just make it 100 not 100%,
   ```
